pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -911,6 +912,28 @@ func LoadSQLDescsFromBackupsAtTime(
 	return allDescs, lastBackupManifest
 }
 
+// FindLatestReIntroductionByTable finds the endtime of the latest incremental
+// backup that reintroduced each backed up table, as of restore time. Note: this
```

Comment:
nit: "endtime of the latest incremental backup that reintroduced each backed up table" confuses me a little because its not really "each backup up table", its only the tables that were reintroduced because of being offline at some point between backups.

Maybe we can say "finds the EndTime of the latest incremental backup for each table that was reintroduced because of being previously offline". See ReintroducedSpans() for more information.